javascript/javascript3/week3/shoppingCard.js

```diff
@@ -0,0 +1,78 @@
+class Product {
+    constructor(name, price) {
+      this.name = name;
+      this.price = price;
+    }
+
+    convertToCurrency(currency) {
+        switch (currency){
+            case "dollars": return this.price *= 0,16; break;
+            case "euro": return this.price *= 0,13; break;
+            case "pound": return this.price *= 0,11; break;
+        }
+    }
```

Comment:
This function wil only return `16`, `13` or `11`. That's because you use commas instead of points.

```suggestion
    convertToCurrency(currency) {
        switch (currency){
            case "dollars": return this.price *= 0.16; break;
            case "euro": return this.price *= 0.13; break;
            case "pound": return this.price *= 0.11; break;
        }
    }
```